examples/test/AttributeFilter_test.js

```diff
@@ -0,0 +1,39 @@
+import { Selector } from 'testcafe';
+import { config } from './utils/config';
+import { loginUsingGreyPages } from './utils/helpers';
+
+fixture('Attribute filter components')
+    .beforeEach(loginUsingGreyPages(`${config.hostname}/attribute-filter-components`))
+    .page(`${config.hostname}/attribute-filter-components`);
+
+test('Dropdown filter', async (t) => {
+    const dropdownButton = Selector('.button.s-employee_name');
+    const clearButton = Selector('.s-clear');
+    const selectAllButton = Selector('.s-select_all');
+    const applyButton = Selector('.s-apply');
+    const attributeFiltersOverlay = Selector('.gd-attribute-filter-overlay');
+    const attributeFilterItems = attributeFiltersOverlay.find('.s-attribute-filter-list-item');
+
+    await t
+        .click(dropdownButton)
+        .expect(dropdownButton.hasClass('is-dropdown-open')).ok()
+        .expect(dropdownButton.hasClass('is-active')).ok()
+        .expect(attributeFilterItems.nth(0).child('input[type=checkbox]').checked).ok()
+        .click(clearButton)
+        .expect(attributeFilterItems.nth(0).child('input[type=checkbox]').checked).notOk()
+        .click(selectAllButton)
+        .expect(attributeFilterItems.nth(0).child('input[type=checkbox]').checked).ok()
+        .click(applyButton)
+        .expect(dropdownButton.hasClass('is-dropdown-open')).notOk()
+        .expect(dropdownButton.hasClass('is-active')).notOk();
+});
+
+test('Custom filter', async (t) => {
```

Comment:
Could you update the title of the test so it's saying clearly what is it testing?